### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by waptik** [akd-io/boil#121] [General]
😱 This is such as great tool! This will save time trying to bootstrap a next app from scratch or using next-app examples + some custom installations.

Found a mention about this great tool on [makerlog telegram group](https://t.me/makerlog/284116)

I usually use nextjs cli with an example(chakra) and install other deps(react-hook form) i need along the way

Congrats!

**Comment by akd-io** [comment]
Thank you so much. I'm so pleased to hear that I achieved my goal, at least for you!

Best of luck! 😊
